app/views/docs/permissions.phtml

```diff
@@ -123,10 +130,14 @@ client
 const databases = new Databases(client, '[DATABASE_ID]');
 
 let promise = databases.createDocument(
-    '[COLLECTION_ID]',
-    {'actorName': 'Chris Evans', 'height': 183},
-    ['role:all'], // Anyone can view this document
-    ['user:5c1f88b42259e'] // You can only grant permissions you own, this must be the current user UID
+    '[COLLECTION_ID]', // Collection ID
+    {'actorName': 'Chris Evans', 'height': 183}, // Data
+    [ // Permissions
+        Permission.read("all"), // Anyone can view this document
+        Permission.edit(["team:writers", "team:admin"]), // Only writers and admins can edit this document
+        Permission.delete("user:5c1f88b42259e") 
```

Comment:
Missing comma. Tho I remember I addressed those in a different PR. Maybe a target branch is wrong? Can we check that please?

Reply:
Probably fixed in the permissions PR